modules/afprog/afprog.c

```diff
@@ -38,6 +38,33 @@
 #include <unistd.h>
 #include <signal.h>
 
+typedef struct _AFProgramReloadInfo
+{
+  LogWriter * writer;
+  pid_t       pid;
+} AFProgramReloadInfo;
+
+static inline void
+terminate_descendants_by_pid(const pid_t pid)
+{
```

Comment:
Is `static` similar to `private`?
If it is, then should I prefix most of the functions in that file?